script/licenses.py

```diff
@@ -84,6 +84,7 @@ def from_dict(cls, data: PackageMetadata) -> PackageDefinition:
     "LGPL-3.0-only",
     "LGPL-3.0-or-later",
     "MIT",
+    "MIT-CMU",
```

Comment:
The license identifier was updated from `HPND` to `MIT-CMU` as that better reflects the current license. See https://github.com/python-pillow/Pillow/pull/8460

https://opensource.org/license/cmu-license